rplint/checks.py

```diff
@@ -285,35 +284,17 @@ def check_line(self, lineno, line):
             )
 
 
-@click.command()
-@click.option(
-    "-l",
-    "--line-length",
-    type=click.INT,
-    default=500,
-    help="Line length to check for.",
-)
-@click.argument(
-    "input_file",
-    type=click.File(mode="r"),
-    required=True,
-)
-def rplint(input_file, line_length):
-    """Checks a Markdown file for common writing errors."""
-    checks = [
-        BadWordsCheck(),
-        LineLengthCheck(line_length),
-        BadPhrasesCheck(),
-        ContractionsCheck(),
-        CodeFormatterCheck(),
-        EndingColonCheck(),
-        CodeBlockOrAlertEndsSectionCheck(),
-        BadLinkAnchorCheck(),
-    ]
-    lines = input_file.readlines()
-    for check in checks:
-        check.run(lines)
-        if check:
-            click.secho(check, fg="red")
-        else:
-            click.secho(f"{check}... Passes!", fg="green")
+class SpacesInLineCheck(LineChecker):
+    def __init__(self):
+        super().__init__()
+        self.title = "Spaces in Line Check"
+
+    def check_line(self, lineno, line):
+        if line.endswith(" \n"):
+            self.register_error(
+                lineno, self.error_format % "trailing whitespace"
+            )
+        if not self.in_code_block and "|" not in line and "  " in line:
```

Comment:
I'm terrible at this one.  I learned to type on a manual typewriter in the days when it was required to put two spaces after periods.  LOTS of muscle memory.  Thanks for adding this!